Seulement dans xVideoServiceThief.slb: Makefile
Seulement dans xVideoServiceThief.slb: bin
Seulement dans xVideoServiceThief.slb: build
diff -aur xVideoServiceThief/src/pakunpak/pakunpak.cpp xVideoServiceThief.slb/src/pakunpak/pakunpak.cpp
--- xVideoServiceThief/src/pakunpak/pakunpak.cpp	2008-01-23 18:56:36.000000000 +0100
+++ xVideoServiceThief.slb/src/pakunpak/pakunpak.cpp	2008-09-26 19:57:47.000000000 +0200
@@ -26,6 +26,7 @@
 */
 
 #include "pakunpak.h"
+#include <string.h>
 
 /* Support functions */
 
